app/models/ticket_request.rb

```diff
@@ -268,11 +268,7 @@ def refund
 
   # calculate the total price for this ticket request
   def price
-    return special_price if special_price
-
-    total = tickets_price
-    total += calculate_addons_price
-    total
+    @price ||= special_price.presence || (tickets_price + calculate_addons_price)
```

Comment:
Need to test the following:

Ticket price gets set to X. 
Special Price gets set to N.
Special Price changes to to M.
Does the price memoization get reset to M?